pkg/components/velero/openebs/openebs.go

```diff
@@ -0,0 +1,160 @@
+// Copyright 2020 The Lokomotive Authors
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+// Package openebs deals with configuring Velero openebs plugin.
+package openebs
+
+import (
+	"fmt"
+
+	"github.com/hashicorp/hcl/v2"
+
+	"github.com/kinvolk/lokomotive/internal"
+)
+
+const indentation = 6
+
+// Configuration contains OpenEBS specific parameters.
+type Configuration struct {
+	Credentials            string                  `hcl:"credentials"`
+	BackupStorageLocation  *BackupStorageLocation  `hcl:"backup_storage_location,block"`
+	VolumeSnapshotLocation *VolumeSnapshotLocation `hcl:"volume_snapshot_location,block"`
+	CredentialsIndented    string
```

Comment:
Actually, if we put indentation in the render function, then we can create a temporary struct there and insert this field there, so we don't need it at all. This way, render function does not have side effects (filling this field).